modules/ListView.js

```diff
@@ -59,10 +59,10 @@
       var childView = this.getChildView(model);
       if (childView) {
         childView.dispose();
-        this.unregisterChildView(childView);
+        this.unregisterTrackedView(childView, { shared: false });
```

Comment:
correct, it is just being explicit, but not required.  unregisterTrackedView(childView); will treat the view as a child, not shared, view.